mbed/mbed.py

```diff
@@ -1618,7 +1618,12 @@ def requirements_contains(self, library_name):
         with open(os.path.join(req_path, req_file), 'r') as f:
             return library_name in f.read()
 
-    def check_requirements(self, show_warning=False):
+    def check_requirements(self, show_warning_only=False):
+        skip_requirements = self.get_cfg("NO_REQUIREMENTS", False)
+        if skip_requirements:
+            action("Skipping installed requirements check due to configuration flag.")
```

Comment:
Could we make this a verbose only print?

Reply:
This is a non standard behavior and it's very easy to forget it. An extra line is not an overkill and it passes the responsibility to the user to handle requirements. Without the notice, there's no indication that one of the fundamental mbed CLI functions is not being performed because of user config.

Reply:
I think having it under a verbose print makes sense, we already tell the user to run with `-v` if the subprocess exits with a non zero code: https://github.com/ARMmbed/mbed-cli/blob/master/mbed/mbed.py#L3388

Reply:
And I already explained that not reminding that we're skipping all requirements check (which is NOT RECOMMENDED) should be clearly visible. Read my comment.